packages/react-instantsearch-hooks-dom/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from 'react-instantsearch-hooks';
```

Comment:
Do we re-export all the hooks from `react-instantsearch-hooks`? I might have misread it, but my understanding of the RFC was to only re-export renderless components (like `<Configure />`).

Reply:
We export everything, so a user doesn't need to add both -hooks and -hooks-dom. This is the same as -core and -dom today :)